selftests/check.py

```diff
@@ -743,16 +748,35 @@ def create_suites(args):  # pylint: disable=W0621
 
         suites.append(TestSuite.from_config(config_check_optional, "optional-plugins"))
 
+    test_dir = "vmimage"
```

Comment:
IMO there is no need to have `vmimage` as variable, but if you want to create `test_dir` variable for easier future changes I would prefer the whole path which would be `os.path.join("selftests", "vmimage")`